### DISCUSSION TITLE
Usage of "propMovement"

### DISCUSSION THREAD

**Discussion by erkhans** [mdolab/dafoam#729] [General]
Hello, Professor. As I understood, we need to use  "propMovement: True " to turn on the 2-way FSI. Is that right? But, when I use it, I get the error shown below. Could you please help to resolve this issue?
![error](https://github.com/user-attachments/assets/0b06a120-6eed-48ce-8a8e-d2f8ef2dba58)

**Comment by friedenhe** [comment]
No, propMovement is optional for 2-way FSI. It is useful only if you have actuator disks in your FSI case.

**Reply by erkhans** [reply]
I am sorry, Professor. I got a little lost. Could you please tell what do we need to do to make 2-way fsi in dafoam using TACS solver?

**Comment by friedenhe** [comment]
This tutorial uses 2-way FSI: https://dafoam.github.io/v3-pages/mydoc_tutorials_aerostruct_mach_wing.html

**Reply by erkhans** [reply]
Thank you, Professor. If we use runPrimal in this tutorial, does it mean that we are running 2-way FSI?

**Reply by friedenhe** [reply]
Yes!

**Reply by erkhans** [reply]
Thank you very much, Professor. Also, is there any possible way to conduct a one-way FSI? I wanted to compare the results.

**Comment by friedenhe** [comment]
No, we do not support one-way FSI..
